eth-brownie/brownie
# Fix: Include explorer support for Optimism and Gnosis chain

### What I did
Just added a few lines to the regex to accept new env vars.  Note that the Optimistim scanner is optimistic.etherscan, so I put that one up top.

Also updated the starting network-config file to include mainnets and forks for the above chains.
Related issue: #

### How I did it
Just added a few lines that I am using locally.

### How to verify it
I'm a forked version actively to operate on these chains and run cross-chain reporting.

I did not see any tests for side chain explorers, so I did not add one.  I also could not find a fitting area to add more to the docs.

### Checklist
